scribejava-apis/src/test/java/com/github/scribejava/apis/examples/Box20Example.java

```diff
@@ -0,0 +1,90 @@
+package com.github.scribejava.apis.examples;
+
+import java.util.Random;
+import java.util.Scanner;
+import com.github.scribejava.core.builder.ServiceBuilder;
+import com.github.scribejava.apis.BoxApi20;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthRequest;
+import com.github.scribejava.core.model.Response;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.oauth.OAuth20Service;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public final class Box20Example {
+
+    private static final String NETWORK_NAME = "Box";
+    private static final String PROTECTED_RESOURCE_URL = "https://app.box.com/profile";
```

Comment:
It's an almost super Pull Request, except for this URL ;-) it's not a protected API URL. It's usual html web page url.
There should be something like this
"https://api.box.com/2.0/users/me"

(or any other demonstrative url from here https://docs.box.com/reference)

It's the single minor problem here. I have fixed it while merging